pkg/apis/kubevirt/v1/types.go

```diff
@@ -71,6 +71,9 @@ type VersionSpec struct {
 
 type ComponentSpec struct {
```

Comment:
Please rename this struct to `NodeLabellerSpec` as it is not used by any other CR and looks confusing

Reply:
@shwetaap You were correct offline, we shouldn't change the name of this struct

Reply:
Why we shouldn't change it?

Reply:
If a change is made here, then it must be updated in HCO as well. I am not sure if that is what we want to do
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/d4c0e35272c4ee7b05f0ecb84dc4cb2549643e09/pkg/controller/hyperconverged/hyperconverged_controller.go#L1307

Reply:
No, we can leave it as is. I was just curious what was the reason.